core/src/main/java/org/mule/runtime/core/internal/profiling/tracing/event/span/NamedSpanBasedOnParentSpanChildSpanCustomizationInfo.java

```diff
@@ -42,4 +48,10 @@ private Optional<InternalSpan> getSpan(CoreEvent coreEvent) {
   public String getLocationAsString(CoreEvent coreEvent) {
     return getSpan(coreEvent).map(internalSpan -> internalSpan.getAttribute(LOCATION_KEY).orElse("")).orElse("");
   }
+
+  @Override
+  public boolean isExportable(CoreEvent coreEvent) {
```

Comment:
Shouldn't this method implementation go on the child class (`NoChildrenExportableUntilExecuteNextSpanCustomizationInfo`)?

Reply:
No, because this is done to avoid exporting a double execute-next:route as the execute next is a processor. But I will refactor this to add it to a different spanCustomizerInfo.

Reply:
I've addressed this comment [in this commit.](https://github.com/mulesoft/mule/pull/11784/commits/987c454578f89317f0bb9df02ade4a2b91ec5403)